api/openapi/v2/core-metadata.yaml

```diff
@@ -420,14 +423,7 @@ components:
           type: string
       required:
       - config
```

Comment:
Remove the `required` section from all response DTOs.

Reply:
Removed.